### PR TITLE
Fix bug in hook definition for openapi-linter

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by ls-samer-abuthaher — APPROVED** [review]

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -318,4 +318,3 @@
   entry: 809245501444.dkr.ecr.eu-central-1.amazonaws.com/openapi-linter:release-latest-main
   language: docker_image
   pass_filenames: false
-  files: '^apidoc/**/{openapi,asyncapi}.yaml$'
```
Comment (ls-jad-elkik) [inline .pre-commit-hooks.yaml:321]: Looks like you want:

```
files: '^apidoc/[^/]+/{openapi,asyncapi}.yaml$'
```

It's a regex not a glob expression.

Comment (ls-jad-elkik) [reply .pre-commit-hooks.yaml:321]: Ideally you'd also have `pass_filenames: true` so it only runs on specific files that change, but idk how easy it is for docker-based pre-commit hooks.

Might not be worth the trouble now at least.

Comment (ls-samer-abuthaher) [reply .pre-commit-hooks.yaml:321]: should that be
```
files: '^apidoc/[^/]+/(openapi|asyncapi).yaml$'
```

Comment (ls-jad-elkik) [reply .pre-commit-hooks.yaml:321]: Yes what Samer said 😅

Comment (ls-jad-elkik) [reply .pre-commit-hooks.yaml:321]: Even better:

```
files: '^apidoc/[^/]+/(openapi|asyncapi)\\.yaml$'
```

**Pull request merged**